src/fdw/trigger.rs

```diff
@@ -93,7 +93,21 @@ unsafe fn auto_create_schema_impl(fcinfo: pg_sys::FunctionCallInfo) -> Result<()
 
     // Get relation name, oid, etc. that triggered the event
     let relation = create_stmt.relation;
-    let schema_name = CStr::from_ptr((*relation).schemaname).to_str()?;
+
+    // Fetch the current schema name if the pointer to it is null
+    // This pointer is null in pg13 when the user doesn't specify the schema in the CREATE FOREIGN TABLE statement
+    let schema_name = match ((*relation).schemaname as *const c_char).is_null() {
+        true => Spi::connect(|client| {
+            client
+                // Casting as TEXT because the default return type is name (Oid(19)) and the conversion to Rust string fails
+                .select("SELECT CURRENT_SCHEMA::TEXT", None, None)?
+                .first()
+                .get_one()
+        })?
+        .unwrap(),
+        false => CStr::from_ptr((*relation).schemaname).to_str()?,
+    };
```

Comment:
This codeblock should probably be put behind a `pg13` feature-flag. It doesn't seem to be required otherwise, no? Something like this

```bash
#[cfg(feature = "pg13")]
```

Reply:
Sounds good. My initial thought was to retain the code to prevent the same issue from happening if the pointer is `null` in future versions of Postgres but that probably won't happen. 
Like this we can see that we can get rid of this code once we deprecate support for pg13 👍

Reply:
Yes exactly! Thank you 🙏